ui/src/app/models/workflowEntry.model.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {WorkflowModel} from "./workflow.model";
+
+export class WorkflowEntryModel {
```

Comment:
Maybe that should be renamed to DynamicFormEntryModel? It's not specific to the workflow

Reply:
Yes, but I would leave it as it is. It is used only in workflow form so that is why I used that name. DynamicFormEntryModel doesn't make sense for me because it is used also in non-dynamic parts. It could be renamed to FormEntryModel, but I would leave it as it is.